recipes/h2mm-c/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "h2mm-c" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "949ab32a3255d286fbb09b2bb9918bac21d4fd6c0cb4216dfc264bf82fa827ce" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/harripd/H2MMpythonlib/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+  host:
+    - python >=3.7
+    - setuptools >=64
+    - setuptools-scm >=6.0
+    - wheel
+    - cython
+    - ipython
+    - numpy
+    - pip
+  run:
+    - python >=3.7
+    - ipython
```

Comment:
```suggestion
```
your code does not import `ipython`.

Avoid the urge to make : `h2mm-c` a convenience package for you.

Reply:
If you check https://github.com/harripd/H2MMpythonlib/blob/main/H2MM_C/H2MM_C.pyx
you will see that the code does import Ipython

I use it specifically to make display in jupyter notebooks convenient. It is an iterative algorithm and so I use the DisplayHandle and Pretty objects to make each iteration display in the same output instead of having a very long list of Iteration 1, Iteration2 ...

Reply:
got it!

Sorry about that, i searched for `ipython` in your source and didn't find it. I should have been case insensitive.

I typically use tqdm
https://github.com/tqdm/tqdm/

Reply:
I looked into tqdm, it didn't quite have all the features I wanted, at least when I was first coding this project.

I wanted a way to report on the current state of the optimization, so I needed to be able to update the display with more arbitrary text than the progress bar tqdm provided.